content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -289,24 +276,31 @@ sudo kubeadm upgrade apply
 
 Also `sudo kubeadm upgrade plan` is not needed.
 
-### Upgrade kubelet and kubectl
+-  Drain the control plane node:
```

Comment:
preview:
https://deploy-preview-24704--kubernetes-io-master-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#upgrade-additional-control-plane-nodes

i think this should look like the following:

```
### Drain control plane nodes and upgrade kubelets

Repeat for all control plane nodes, one at a time.

Drain the control plane node:

....

Upgrade the kubelet:

...

Uncordon the node:

...

### Upgrade kubectl

...

```